src/build/get_root_dir.mjs

```diff
@@ -17,7 +17,11 @@ import url from 'url';
 
 // WARNING: if you move this file, you MUST update this file path
 const RELATIVE_PATH_FROM_THIS_FILE_TO_PROJECT_ROOT = '../..';
```

Comment:
Does this need to use path join?

Reply:
I use path.resolve just below

Reply:
I believe path.resolve would still expect the right separator.